roles/matrix-bot-postmoogle/templates/systemd/matrix-bot-postmoogle.service.j2

```diff
@@ -24,7 +24,9 @@ ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-bot-postmoogle
 			--network={{ matrix_docker_network }} \
 			--env-file={{ matrix_bot_postmoogle_config_path }}/env \
 			-p {{ matrix_bot_postmoogle_smtp_host_bind_port }}:{{ matrix_bot_postmoogle_port }} \
+			-p {{ matrix_bot_postmoogle_smtps_host_bind_port }}:{{ matrix_bot_postmoogle_tls_port }} \
 			--mount type=bind,src={{ matrix_bot_postmoogle_data_path }},dst=/data \
+			--mount type=bind,src={{ matrix_bot_postmoogle_ssl_path }},dst=/ssl \
```

Comment:
```suggestion
			{% if matrix_bot_postmoogle_ssl_path %}
			--mount type=bind,src={{ matrix_bot_postmoogle_ssl_path }},dst=/ssl \
			{% endif %}
```

If `matrix_bot_postmoogle_ssl_path` defaults to an empty value (as suggested by my other review comment), we need to ensure we don't try to mount it.